docs/release-notes/mia-platform-v13.mdx

```diff
@@ -7,3 +7,140 @@ image: "img/release-note-link-preview.png"
 
 **Welcome to Mia-Platform v13!**
 
+# Compose your Platform
+With our latest release, we empower you to "Compose your Platform," offering the freedom to compose your platform just the way you envision it. Whether you're forging new paths or refining existing strategies, Mia-Platform v13 is here to support you every step of the way, giving you the flexibility and control you need to craft a solution that fits you.
+Let's take a closer look at all the new features included in Mia-Platform v13.
+
+## Enhance Extensibility
+
+In today's fast-paced tech landscape, adaptability is key. That's why Mia-Platform v13 takes extensibility to the next level, making it easier than ever to integrate third-party tools and custom functionalities into your platform. We welcome **Platforge**, our brand new Extensibility toolkit, in the Mia-Platform eco-system!
+The main benefits for the user can be summarized by the following:
+
+- Extend developer lifecycle to be aligned with the needs of each team (Extend supported runtime, extend dev workflow, extend application management…).
+- Brownfield can standardize and simplify their developer workflow.
```

Comment:
```suggestion
- Brownfield users can standardize and simplify their developer workflow.
```

Reply:
Are we sure that 'Brownfields' is a term familiar enough to our readers?

Reply:
yeah you may be right, we'll try to rephrase it